examples/validation/validate_grid1.py

```diff
@@ -0,0 +1,119 @@
+"""
+Validate the grid
+=================
+
+Show a bunch of grids touching various grid-cases.
+
+"""
+
+# sphinx_gallery_pygfx_docs = 'screenshot'
+# sphinx_gallery_pygfx_test = 'compare'
+
+from wgpu.gui.auto import WgpuCanvas, run
+import pygfx as gfx
+
+
+canvas = WgpuCanvas()
+renderer = gfx.renderers.WgpuRenderer(canvas)
+scene = gfx.Scene()
+scene.add(gfx.Background.from_color("#fff"))
+
+
+# An inf grid at the floor
+grid0 = gfx.Grid(
+    None,
+    gfx.GridMaterial(
+        thickness_space="world",
+        axis_thickness=0.1,
+        major_thickness=0.02,
+        minor_thickness=0.01,
+        inf_grid=True,
+    ),
+    orientation="xz",
+)
+
+# A grid oriented in yz, showing major and minor ticks
+grid1 = gfx.Grid(
+    None,
+    gfx.GridMaterial(
+        major_step=1,
```

Comment:
Are the steps in thickness_space too?

Reply:
If the grid is aligned with a world axis, the grid' lines are always exactly on "whole steps" in world-space, regardless of scale or offset. See how the lines on the green and blue grid touch.

Reply:
I guess this seems to make sense right? For example, the grid lines would correspond to data in the positions buffer of a line?

Reply:
OK, that makes sense!